src/i18n/strings/en_EN.json

```diff
@@ -1767,6 +1770,32 @@
     "%(brand)s is end-to-end encrypted, but is currently limited to smaller numbers of users.": "%(brand)s is end-to-end encrypted, but is currently limited to smaller numbers of users.",
     "You do not have sufficient permissions to change this.": "You do not have sufficient permissions to change this.",
     "Call type": "Call type",
+    "Email summary": "Email summary",
+    "Receive an email summary of missed notifications": "Receive an email summary of missed notifications",
+    "Select which emails you want to send summaries to. Manage your emails in <button>General</button>.": "Select which emails you want to send summaries to. Manage your emails in <button>General</button>.",
+    "People, Mentions and Keywords": "People, Mentions and Keywords",
+    "Mentions and Keywords only": "Mentions and Keywords only",
+    "Switch now": "Switch now",
+    "<strong>Update:</strong> We have updated our notification settings. This won’t affect your previously selected settings.": "<strong>Update:</strong> We have updated our notification settings. This won’t affect your previously selected settings.",
+    "Show message preview in desktop notification": "Show message preview in desktop notification",
+    "I want to be notified for (Default Setting)": "I want to be notified for (Default Setting)",
+    "This setting will be applied by default to all your rooms.": "This setting will be applied by default to all your rooms.",
+    "Play a sound for": "Play a sound for",
+    "Applied by default to all rooms on all devices.": "Applied by default to all rooms on all devices.",
+    "Mentions and Keywords": "Mentions and Keywords",
+    "Audio and Video calls": "Audio and Video calls",
+    "Other things we think you might be interested in:": "Other things we think you might be interested in:",
+    "Invited to a room": "Invited to a room",
+    "New room activity, upgrades and status messages occur": "New room activity, upgrades and status messages occur",
+    "Messages are sent by a bot": "Messages are sent by a bot",
```

Comment:
This reads weirdly IMO

Reply:
Can you be more specific? Can I also ask that we have another PR to handle any future content changes as this copy is as per the figma

Reply:
<img width="371" alt="image" src="https://github.com/matrix-org/matrix-react-sdk/assets/2403652/b839de5f-99c1-4aa8-afb2-440497646957">

I don't think it reads right, `Messages *THAT* are sent by a bot` maybe?

Reply:
Or simpler yet `Messages sent by bots`